engine/CMakeLists.txt

```diff
@@ -85,6 +85,8 @@ add_executable(cloe-engine
     src/coordinator.cpp
     src/simulation.cpp
     src/simulation_context.cpp
+    src/simulation_context.hpp
+    src/simulation_progress.hpp
```

Comment:
.hpp files in add_executable?

Reply:
They are for IDE / LSP support. For the build itself CMake ignores them.